cylc/flow/xtriggers/echo.py

```diff
@@ -31,7 +30,14 @@ def echo(*args, **kwargs):
     """
     print("echo: ARGS:", args)
     print("echo: KWARGS:", kwargs)
-    result = False
-    with suppress(KeyError):
-        result = kwargs["succeed"] is True
-    return result, kwargs
+
+    return kwargs["succeed"], kwargs
+
+
+def validate(f_args, f_kwargs, f_signature):
+    try:
+        assert type(f_kwargs["succeed"]) is bool
+    except (KeyError, AssertionError):
```

Comment:
Bandit complaining about `assert` 
```suggestion
    if "succeed" not in f_kwargs or not type(f_kwargs["succeed"]) is bool:
```

Reply:
oops, yes